java/com/google/devrel/gmscore/tools/apk/arsc/Chunk.java

```diff
@@ -17,24 +17,25 @@
 package com.google.devrel.gmscore.tools.apk.arsc;
 
 import com.google.common.base.Preconditions;
-import com.google.common.collect.ImmutableMap;
-import com.google.common.collect.ImmutableMap.Builder;
+import com.google.common.io.Closeables;
 import com.google.common.io.LittleEndianDataOutputStream;
 import com.google.common.primitives.Shorts;
-
 import java.io.ByteArrayOutputStream;
 import java.io.DataOutput;
 import java.io.IOException;
 import java.nio.ByteBuffer;
 import java.nio.ByteOrder;
+import java.util.Collections;
+import java.util.HashMap;
 import java.util.Map;
-
 import javax.annotation.Nullable;
 
 /** Represents a generic chunk. */
 public abstract class Chunk implements SerializableResource {
 
   /** Types of chunks that can exist. */
+  // See
+  // https://cs.corp.google.com/aosp-master/frameworks/base/libs/androidfw/include/androidfw/ResourceTypes.h?rcl=2b1b1bb22dcab2467f8a41825f36d6be68a031f5&l=215
```

Comment:
please scrub internal link or convert it to an external one if applicable

Reply:
changed to aosp link